gbpservice/nfp/service_vendor_agents/vyos/agent/src/vyos_init_script/restart_vpn

```diff
@@ -0,0 +1,16 @@
+#!/bin/vbash
+
```

Comment:
- Retained vyos_init_script directory and removed init_script directory. 

- Removed vyos file from vyos_init_script directory as the same was available in etc/init.d directory

@maheshkurund Please review. @njagadish